### DISCUSSION TITLE
Does Applite leave an orphaned homebrew instance on uninstall?

### DISCUSSION THREAD

**Discussion by Younghart** [milanvarady/Applite#44] [General]
Hi,

I installed Applite on a new machine without an existing homebrew installation. Later on, I uninstalled Applite from within the Applite App. Afterwards, I looked in the ~/Library/ Application Support/Applite/homebrew directory, and there still was a full homebrew installation.
Does this mean that the deinstallation process leaves the homebrew installation intact and active, even when homebrew was installed by Applite? I think this might be a security risk, especially for those who aren't familiar with package managers.

**Comment by milanvarady** [comment]
Moved to issue #45
